web/src/user_profile.js

```diff
@@ -96,6 +96,11 @@ export function update_user_profile_streams_list_for_users(user_ids) {
     }
 }
 
+export function update_bot_data(bot_id) {
```

Comment:
Would be good to rename it something like `update_bot_profile_modal_ui` to make it more clear.

Reply:
Fixed it.